factory/processing/blockProcessorCreator.go

```diff
@@ -451,6 +458,39 @@ func (pcf *processComponentsFactory) newShardBlockProcessor(
 	}, nil
 }
 
+func (pcf *processComponentsFactory) addSystemVMToContainer(vmContainer process.VirtualMachinesContainer, builtInFuncFactory vmcommon.BuiltInFunctionFactory) error {
```

Comment:
Unit tests for this method and the new code added in L169!

Reply:
added